src/pixelator/graph/backends/implementations.py

```diff
@@ -456,17 +654,36 @@ def from_edgelist(
         :rtype: NetworkXGraphBackend
         :raises: AssertionError when the input edge list is not valid
         """
-        raise NotImplementedError()
+        # TODO If we could change the signature here to work with lazy frames
```

Comment:
pl.LazyFrame all the way?

Reply:
I forgot to remove this comment, I did implement it like this. I'll remove the comment.